src/org/ods/orchestration/service/JiraService.groovy

```diff
@@ -210,20 +210,20 @@ class JiraService {
     }
 
     @NonCPS
-    Map createIssueType(String type, String projectKey, String summary, String description, String fixVersion = null) {
-        if (!type?.trim()) {
+    Map createIssue(Map args, String fixVersion = null, String component = null, String priority = null) {
```

Comment:
Please make all arguments named arguments. Mixing positional and named arguments is confusing. In this case, the method should only take a map.

Reply:
Done.

Reply:
Fine now